src/System Application/App/Entity Text/src/EntityTextImpl.Codeunit.al

```diff
@@ -202,12 +221,35 @@ codeunit 2012 "Entity Text Impl."
 
     [NonDebuggable]
     local procedure GetPromptInfo(): JsonObject
+    var
+        PromptSecretName: Text;
+    begin
+        PromptSecretName := PromptObjectKeyTxt;
+
+        exit(GetPromptObject(PromptSecretName))
+    end;
+
+    [NonDebuggable]
+    local procedure GetPromptInfo(SourceScenarioName: Text): JsonObject
+    var
+        PromptSecretName: Text;
+    begin
+        if SourceScenarioName = Format(Enum::"Entity Text Scenario"::"Reminder Text") then
```

Comment:
Same as before.

Reply:
Same answer as before - it is a conscious decision not to raise events, but hardcode Reminder Text inside Entity Text. We must not let subscribers provide their own prompt.